components/molecule/photoUploader/src/fileTools.js

```diff
@@ -2,53 +2,78 @@ import {formatToBase64} from './photoTools'
 
 import {
   DEFAULT_HAS_ERRORS_STATUS,
-  DEFAULT_IMAGE_ROTATION_DEGREES
+  DEFAULT_IMAGE_ROTATION_DEGREES,
+  REJECT_FILES_REASONS
 } from './config'
 
 export const filterValidFiles = ({
   files,
   filesToBeFiltered,
   acceptedFileMaxSize,
+  callbackPhotosRejected,
```

Comment:
Maybe you can use a naming with handle: `handlePhotosReject`

Reply:
Renamed! :)